content/release-10-16-0/applications-10-16-0-bundle/10-16-0-0.md

```diff
@@ -23,24 +23,47 @@ Issue</th>
 <tr>
 <td>
 Java SDK</td>
-<td> The following Maven dependency changes have been made in the `cumulocity-clients-java` SDK:
-<br>- the internal model related dependencies with dedicated `java-client-model` have been replaced
-<br>- a new `svenson` dependency dedicated for the SDK has been introduced which replaces the internal one
-<br>- the javax validation dependency has been updated to `jakarta.validation-api` version 2.0.2
-<br>If your microservice does not define explicit dependencies to the `core-model` or `application-model` no additional work should be required, otherwise you should remove the above dependencies and replace them with `java-client-model`.
+<td> The following Maven dependency changes have been made in the <code>cumulocity-clients-java</code> SDK:
+<br>- the internal model related dependencies with dedicated <code>java-client-model</code> have been replaced
+<br>- a new svenson dependency dedicated for the SDK has been introduced which replaces the internal one
+<br>- the javax validation dependency has been updated to <code>jakarta.validation-api</code> version 2.0.2
+<br>If your microservice does not define explicit dependencies to the <code>core-model</code> or <code>application-model</code> no additional work should be required, otherwise you should remove the above dependencies and replace them with <code>java-client-model</code>.
 <br>If your microservice already defines the javax validation dependency adjust the  version to the one from the SDK. </td>
 <td>
 MTM-46315</td>
 </tr>
 
 <tr>
 <td>
- Web SDK </td>
-<td> If tracking the user experience is enabled for a user, the Knowledge Hub button used to show up automatically in the UI. Users can now enable or disable the Knowledge Hub in the user settings separately. </td>
+Web SDK </td>
+<td> If tracking the user experience is enabled for a user, the <b>Knowledge Hub</b> button used to show up automatically in the UI. Users can now enable or disable the Knowledge Hub in the user settings separately. </td>
 <td>
 MTM-45735</td>
 </tr>
 
+<tr>
+<td>
+Web SDK</td>
+<td>The Web SDK has been upgraded to Angular 14.</td>
+<td>
+MTM-43453</td>
+</tr>
+
+<tr>
+<td>
+Web SDK</td>
+<td>Added a <b>Grids</b> > <b>Device grid</b> tab in the Tutorial application to present usage examples for the DeviceGridComponent.</td>
```

Comment:
Will this render correctly? there's a single ">" between "Grids" and "Device grid", should it not be escaped?

Reply:
Yes, maybe you are right, will do.